components/datetime/src/pattern/runtime/plural.rs

```diff
@@ -14,24 +14,28 @@ use icu_provider::{yoke, zerofrom};
 
 /// A collection of plural variants of a pattern.
 #[derive(Debug, PartialEq, Clone, yoke::Yokeable, zerofrom::ZeroFrom)]
-#[cfg_attr(feature = "datagen", derive(serde::Serialize))]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, crabbake::Bakeable),
+    crabbake(path = icu_datetime::pattern::runtime),
+)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 pub struct PluralPattern<'data> {
     /// The field that 'variants' are predicated on.
-    pivot_field: Week,
+    pub pivot_field: Week,
```

Comment:
Question: Please verify that there are no invariants in this struct

Reply:
This has a derived `Deserialize`...